lunar/distribution.yaml

```diff
@@ -1,6 +1,6 @@
 %YAML 1.1
 # ROS distribution file
-# see REP 143: http://ros.org/reps/rep-0143.html
+# see REP 141: http://ros.org/reps/rep-0141.html
```

Comment:
Your version of `python-rosdistro` is out of date. Please update to the latest release. For this PR either manually change the patch to not modify the comment line at the top of the file or create a new patch using `bloom-release ... -p` after updating and close this one.